src/modules/fancyzones/editor/FancyZonesEditor/Utils/FancyZonesEditorIO.cs

```diff
@@ -0,0 +1,678 @@
+// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.IO;
+using System.IO.Abstractions;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using System.Windows;
+using FancyZonesEditor.Models;
+
+namespace FancyZonesEditor.Utils
+{
+    public class FancyZonesEditorIO
+    {
+        // JSON tags
```

Comment:
Let's add a comment that all the const strings in this class are IDs and should not be localized.